lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -173,9 +173,13 @@
     }
   }
 
+  .ccx-grade-book-download {
+    text-align: right;
+  }
+
   h1 {
     @extend .top-header;
-    display: inline-block;
+    display: block;
```

Comment:
@pdpinch  this is to fix the grade book heading, because of previous code I was getting error in adjusting grade book box inside student admin page.